src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -393,6 +420,13 @@ - (void)setFrame:(CGRect)frame {
   [self layoutVisiblePages];
 }
 
+- (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event {
```

Comment:
Add a comment explaining why we have to do this - otherwise future maintainer will have to figure it out themselves :(